.gitignore

```diff
@@ -26,3 +26,7 @@ releases/
 .idea/modules/
 .idea/libraries/Gradle_*.xml
 .idea/modules.xml
+flutter-studio/testSrc/config
```

Comment:
Just to confirm, we don't want this source checked in?

Reply:
TBH I have not looked at it. It is a cache created by the GUI test framework. I just assumed that others would prefer to not have it taking up space.

Thanks for the review!